internal/status/conditions.go

```diff
@@ -210,6 +216,123 @@ func (c *Cache) getGatewayConditions(gatewayStatus []gatewayapi_v1alpha1.RouteGa
 	return map[gatewayapi_v1alpha1.RouteConditionType]metav1.Condition{}
 }
 
+type GatewayConditionsUpdate struct {
```

Comment:
If this is staying, then maybe move it to its own go file and rename this file to `routeconditions.go`?

Reply:
Commit `4bd950a` splits route/gc/gateway conditions into separate files.